### ISSUE TITLE
Inference for Wav2vec 2.0, how to set --wer-args '("/path/to/lm/4-gram.bin","/path/to/lexicon",2,-1)'

### ISSUE THREAD

**Issue by Jxu-Thu** [pytorch/fairseq#3667]
Thanks for your great codes. 
I am reproducing the results of wav2vec 2.0.
For fairsep version 10.2 and master, I find the in training scripts
--wer-args '("/path/to/lm/4-gram.bin","/path/to/lexicon",2,-1)'.

I find many pre-trained models in the link https://github.com/flashlight/wav2letter/tree/master/recipes/sota/2019. But
/path/to/lm/4-gram.bin means which file?
For 4-gram, it refers to which file,
For Transformer LM, it refers to which file?
And /path/to/lexicon refers to which file?

**Different files correspond to which result of your paper?**
For inference, --lm-model /path/to/kenlm.bin,

/path/to/kenlm.bin refers to which file?
I am confused with these paths and do not know what it means.

Hope for more clear instructions.

Another question is that for inference
python examples/speech_recognition/infer.py /checkpoint/abaevski/data/speech/libri/10h/wav2vec/raw --task audio_pretraining \
--nbest 1 --path /path/to/model --gen-subset $subset --results-path /path/to/save/results/for/sclite --w2l-decoder kenlm \
--lm-model /path/to/kenlm.bin --lm-weight 2 --word-score -1 --sil-weight 0 --criterion ctc --labels ltr --max-tokens 4000000 \
--post-process letter

/checkpoint/abaevski/data/speech/libri/10h/wav2vec/raw means what? 
it is the raw data of test-clean/ test-other?
